pkg/storage/client_raft_test.go

```diff
@@ -2363,6 +2363,16 @@ func TestReplicateReAddAfterDown(t *testing.T) {
 	mtc := startMultiTestContext(t, 3)
 	defer mtc.Stop()
 
+	// Disable the raft transport circuit breakers from ever tripping in order to
+	// prevent errors from when a store is down from causing a connection attempt
+	// to get rejected after the store is back up. This is needed because
+	// mtc.replicateRange calls log.Fatal if any single attempt to send an
+	// ADD_REPLICA request fails.
+	downedStoreIdx := 2
+	for _, transport := range mtc.transports {
+		transport.GetCircuitBreaker(roachpb.NodeID(downedStoreIdx + 1)).ShouldTrip = nil
+	}
```

Comment:
Yeah, I was wondering that too. I played around with doing it for resetting circuit breakers before realizing that resetting them wasn't enough. I've moved this back into a helper.